src/builtins/status.rs

```diff
@@ -510,6 +486,55 @@ pub fn status(parser: &Parser, streams: &mut IoStreams, args: &mut [&wstr]) -> B
             streams.out.append(src);
             return Ok(SUCCESS);
         }
+        STATUS_LOCALE => {
+            cfg_if! {
+                if #[cfg(not(feature = "localize-messages"))]
+                {
+                    streams.err.append(L!("fish was built with the `localize-messages` feature disabled. The `status locale` command is unavailable.\n"));
+                    return Err(STATUS_CMD_ERROR);
+                } else {
```

Comment:
the else after return is a bit ugly,
if it's implied by `cfg_if`, that's another (superficial) reason for moving 
from `#[cfg(not(feature = "localize-messages"))]` to `if !cfg(feature = "localize-messages")`